.eslintrc

```diff
@@ -0,0 +1,51 @@
+{
+  "extends" : [
+    "standard",
+    "plugin:promise/recommended",
+  ],
+  "plugins": [
+    "promise"
+  ],
+  "env": {
+    "browser" : true,
+    "node"    : true,
+    "mocha"   : true,
+    "jest"    : true
+  },
+  "globals" : {
+    "artifacts": false,
+    "contract": false,
+    "assert": false,
+    "web3": false
+  },
+  "rules": {
+
+    // Strict mode
+    "strict": [2, "global"],
+
+    // Code style
+    "indent": [2, 2],
+    "quotes": [2, "single"],
+    "semi": ["error", "always"],
```

Comment:
@facuspagnuolo the discussion about this is here: https://github.com/ZeppelinSolutions/code-style/issues/1
I might have misunderstood your opinion there, sorry about that. Feel free to reopen it and we'll keep talking there.
Later, if we agree to not use the semicolons, I'll adjust the code. This one is super simple, because eslint can autoapply it.